tests/e2e/conformance_test.go

```diff
@@ -93,12 +93,16 @@ func (e *fsxDriver) CreateVolume(config *testsuites.PerTestConfig, volType testp
 	}
 	securityGroupIds := getSecurityGroupIds(instance)
 	subnetId := *instance.SubnetId
+	defaultStorageType := "HDD"
+	defaultDriveCacheType := "NONE"
 
 	ctx := context.Background()
 	options := &fsx.FileSystemOptions{
 		CapacityGiB:      3600,
 		SubnetId:         subnetId,
 		SecurityGroupIds: securityGroupIds,
+		StorageType:      defaultStorageType,
```

Comment:
per my latest comment including the cloudtrail message, also need to set deploymentType to PERSISTENT_1?

Reply:
I included this line of code due to a previous test failure. I realize now that the previous test failure was related to a bug in another code path, so I've gone ahead and removed this parameter in the latest revision.